src/TMedia.cpp

```diff
@@ -657,7 +683,7 @@ TMediaPlayer TMedia::matchMediaPlayer(TMediaData& mediaData, const QString& abso
 #if (QT_VERSION) >= (QT_VERSION_CHECK(5, 14, 0))
             if (pTestPlayer.getMediaPlayer()->media().request().url().toString().endsWith(absolutePathFileName)) { // Is the same sound or music playing?
 #else
-            if (pTestPlayer.getMediaPlayer()->media().canonicalUrl().toString().endsWith(absolutePathFileName)) { // Is the same sound or music playing?
+            if (pTestPlayer.getMediaPlayer()->media().canonicalUrl().toString().endsWith(absolutePathFileName)) {  // Is the same sound or music playing?
```

Comment:
this seems like an accidental space?

Reply:
@demonnic I tried to remove it too, but apparently the formatter that Mudlet drives into vscode adds it - perhaps aligning with the comment above it?  Can it be that smart?  The code formatter made another change elsewhere too that was not of my doing. I could take this space out from an edit on the command line before I check-in if we really believe it is needed, but I suspect that it will come back upon the next edit by myself or someone else using vscode.

Reply:
If it is, that would be https://github.com/Mudlet/Mudlet/blob/development/src/.clang-format, though this is fine either way I'd say

Reply:
Yeah not a huge issue either way, but was literally the only question I had so I figured I'd ask it.